data/hai/hai reveal 4 middle.txt

```diff
@@ -1417,25 +1417,59 @@ mission "Hai Reveal [A14] A Time of Trouble"
-			`	Xilin nods grimly. "However, that would be a drastic failure of foreign policy. It would undermine the legitimacy of the Republic, especially after the war we've just had, and it would be impossible to realistically assure people that an invasion is really not going to happen. True or not. In all likelihood The Republic would fracture and the Free Worlds would probably be on the receiving end of a migration crisis. Parliament has tried to say that they'll take care of it themselves, but the Hai say they've had enough of sitting around and want to be there to save their own people.`
-			`	"They haven't outright said it... but they've implied that the Navy could end up killing the Hai involved in the process by not understanding the capabilities of the Hai, or by placing too great an emphasis on preserving Navy lives. Enough people in the Cabinet disagree, that we're sort of at an impasse, for the moment."`
-			`	"Who in the Cabinet is blocking it?" The speaker is Turner, and everyone turns to look at him with a mixture of irritation or surprise.`
+			`	Xilin nods grimly. "However, that would be a drastic failure of foreign policy."`
+				goto questions
+			label "let hai through"
+			`	Xilin replies, "That would undermine the legitimacy of the Republic, especially after the war we've just had, and it would be impossible to realistically assure people that an invasion is really not going to happen. True or not. In all likelihood the Republic would fracture and the Free Worlds would probably be on the receiving end of a migration crisis. Parliament has tried to say that they'll take care of it themselves, but the Hai say they've had enough of sitting around and want to be there to save their own people.`
+				goto questions
+			label "hai concerns"
+			`	Xilin says, "They haven't outright said it... but they've implied that the Navy could end up killing the Hai involved in the process by not understanding the capabilities of the Hai, or by placing too great an emphasis on preserving Navy lives. Cabinet is in disagreement about what to do, so there is deadlock at the moment."`
+				goto questions
+			label "who is blocking"
 			`	Xilin proceeds to list off a number of names, but after a few Turner cuts in again. "Remington, I know you're the PR guy, and you keep your nose out of sales, so I apologize if it's presumptuous for me to assume you haven't considered this, but, most of those names are people who listen to the Syndicate. I should know, I've had to negotiate with them as a bloc often enough for government contracts. Has anyone on your side reported back to the Syndicate Board and suggested that perhaps, if we can influence them to support a solution that we propose, even if we don't have that solution defined yet, the Hai could be a very large and very profitable market for Syndicate goods?"`
```

Comment:
```suggestion
			`	Xilin proceeds to list off a number of names, but to everyone's irritation and surprise, Turner cuts in. "Remington, I know you're the PR guy, and you keep your nose out of sales, so I apologize if it's presumptuous for me to assume you haven't considered this, but, most of those names are people who listen to the Syndicate. I should know, I've had to negotiate with them as a bloc often enough for government contracts. Has anyone on your side reported back to the Syndicate Board and suggested that perhaps, if we can influence them to support a solution that we propose, even if we don't have that solution defined yet, the Hai could be a very large and very profitable market for Syndicate goods?"`
```
I dunno if this works well, but it changes a part that doesn't make sense anymore.

Reply:
Yes, I need to revisit that section. Also, Captain Last needs to sound less robotic in her responses.

Reply:
With your changes, it does seem okay. (I always test these in game, not by reading the diffs, since everything feels different in the game than in a text editor.) I think the remaining issues with my A14 changes are that Captain Last is robotic.